perception/navigator_vision/nodes/Dock_Detection.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+import cv2
+import numpy as np
+import math
+import argparse
+import imutils
+import rospy
+from matplotlib import pyplot as plt
+from std_msgs.msg import String
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+import roslib
+import sys
+
+# import template picture form local file
+imgCr = cv2.imread('/home/y21/mil_ws/src/NaviGator/perception/navigator_vision/templates/dock_shape_identification/cross.png', 0)
```

Comment:
We need to be able to run your program too... so these files need to be in the repo